scripts/create-clog.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+#
+# create-clog.sh <version> <release-date>
+#
+# Dependencies: sed
+# 
+# Description: Uses template_changelog.md to create a new changelog.
+#
+# Examples:
+#  ./scripts/create-clog.sh "1.18.0" "05/18/2021"
+
+
+set -eou pipefail
+
+VERSION_DELIM="<% X.Y.Z %>"
+RELEASE_DATE_DELIM="<% MM/DD/YYYY %>"
+VERSION=""
+RELEASE_DATE=""
+
+# usage prints information for how to use this program
+usage () {
```

Comment:
```suggestion
function usage () {
```

just a personal preference but for bash-only scripts, the function keyword makes it clearer that this is a function: https://stackoverflow.com/questions/7917018/what-is-the-function-keyword-used-in-some-bash-scripts

we try to use that consistently in other parts of our bash scripts

Reply:
Ah, yes - great call